	Build fix for Cygwin.

	Since <windows.h> is not included for Cygwin, we
	can't use the WIN32 API.

	With U_ENABLE_DYLOAD=1: enable plugins for Cygwin.

	With U_ENABLE_DYLOAD=0: can't use WIN32 API (HMODULE undeclared).

diff -ur icu-49.1.orig/source/common/putil.cpp icu-49.1/source/common/putil.cpp
--- icu-49.1.orig/source/common/putil.cpp	2012-03-21 18:35:10.000000000 +0100
+++ icu-49.1/source/common/putil.cpp	2012-03-23 18:37:30.000000000 +0100
@@ -2100,7 +2100,7 @@
 
 #if U_ENABLE_DYLOAD
  
-#if HAVE_DLOPEN && !U_PLATFORM_HAS_WIN32_API
+#if HAVE_DLOPEN && !U_PLATFORM_USES_ONLY_WIN32_API
 
 #if HAVE_DLFCN_H
 
@@ -2181,7 +2181,7 @@
 
 #endif
 
-#elif U_PLATFORM_HAS_WIN32_API
+#elif U_PLATFORM_USES_ONLY_WIN32_API
 
 U_INTERNAL void * U_EXPORT2
 uprv_dl_open(const char *libName, UErrorCode *status) {
